exam_2.py

```diff
@@ -0,0 +1,6 @@
+Dict={}
+a=input("Bir değer giriniz:")
+for i in range(1,int(a)):

```

Comment:
range() fonksiyonunun ikinci argumani sayi araligina dahil kabul edilmez. Bu yuzden input ile alinan sayinin 1 fazlasini yazmaliyiz. 1.soruda oldugu gibi.

```suggestion
for i in range(1,int(a+1)):
```